substrate-archive/src/actors/workers/storage_aggregator.rs

```diff
@@ -66,6 +82,9 @@ where
 				if addr.send(SendStorage).await.is_err() {
 					break;
 				}
+				if addr.send(SendTraces).await.is_err() {
+					break;
```

Comment:
They won't see anything, however our send can only fail in one way, and that's if StorageAggregator itself disconnects due to some other error: https://docs.rs/xtra/0.5.0-rc.1/xtra/address/struct.SendFuture.html

So this break/error check is to stop the task from running if the actor has shutdown

Reply:
This is fine as long as users get a proper log message when postgres disconnects unexpectedly (because it was shut down or the network connection died); it sounds like you're saying this would be logged in the `StorageAggregator`, correct?

Reply:
Yeah, if something happens with Postgres it would be logged with the database actor

Since `StorageAggregator` is mostly meant as a way to optimize our inserts by gathering bulk data, the errors which can occur are pretty minimal, so mostly anything logged here would be due to actor disconnects external to the Aggregator, which would be logged